crypto/rsa/rsa_gen.c

```diff
@@ -143,6 +144,7 @@ static int rsa_builtin_keygen(RSA *rsa, int bits, int primes, BIGNUM *e_value,
     if (BN_copy(rsa->e, e_value) == NULL)
         goto err;
 
+    BN_set_flags(rsa->e, BN_FLG_CONSTTIME);
```

Comment:
`rsa->e` is a public value, so marking it as constant-time is probably undesirable. This means public RSA operations will use the slower exponentiation algorithm that is meant to hide the exponent.